maat-court-data-api/src/test/java/gov/uk/courtdata/dces/controller/ContributionFileControllerTest.java

```diff
@@ -0,0 +1,135 @@
+package gov.uk.courtdata.dces.controller;
+
+import gov.uk.courtdata.builder.TestEntityDataBuilder;
+import gov.uk.courtdata.dces.mapper.ContributionFileMapper;
+import gov.uk.courtdata.dces.mapper.ContributionFileMapperImpl;
+import gov.uk.courtdata.dces.service.ContributionFileService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@WebMvcTest(ContributionFileController.class)
+@AutoConfigureMockMvc(addFilters = false)
+class ContributionFileControllerTest {
+    private static final String BASE_URL = "/api/internal/v1/debt-collection-enforcement/contribution-file";
+
+    private final ContributionFileMapper mapper = new ContributionFileMapperImpl();
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private ContributionFileService contributionFileService;
+
+    @Test
+    void givenCorrectParameters_whenGetContributionFileInvoked_thenResponseIsReturned() throws Exception {
+        final int fileId = 99;
+        final var response = mapper.toContributionFileResponse(TestEntityDataBuilder.getPopulatedContributionFilesEntity(fileId));
+        when(contributionFileService.getContributionFile(fileId)).thenReturn(Optional.of(response));
+
+        mockMvc.perform(MockMvcRequestBuilders.get(BASE_URL + "/{fileId}", fileId))
+                .andExpect(status().isOk())
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(jsonPath("$.id").value(fileId));
+        verify(contributionFileService).getContributionFile(fileId);
+    }
+
+    @Test
+    void givenIncorrectParameters_whenGetContributionFileInvoked_thenNotFound() throws Exception {
+        final int INCORRECT_FILE_ID = 666;
```

Comment:
Since this is not a constant it should be camel case `INCORRECT_FILE_ID`.

Reply:
all done